test/components/converters/test_openapi_functions.py

```diff
@@ -187,7 +187,7 @@ def test_run_with_bytestream_source(self, json_serperdev_openapi_spec):
         assert (
             doc.content
             == '{"name": "search", "description": "Search the web with Google", "parameters": {"type": "object", '
-            '"properties": {"requestBody": {"type": "object", "properties": {"q": {"type": "string"}}}}}}'
```

Comment:
@julian-risch this is the flatter structure mentioned in https://github.com/deepset-ai/haystack/pull/7034#issue-2142137813 We don't distinguish between URL/query params and requestBody as we did before. We did that because I didn't know how to do OpenConnectorService (195-208) before. Now both URL and requestBody params in OpenAPI spec become simply properties in OpenAI FC spec. We made our OpenConnectorService slightly "smarter". See loc 195-208 of OpenConnectorService. We sort parameters into the right REST API call position there.